validation/src/main/scala/hmda/validation/dsl/PredicateDefaults.scala

```diff
@@ -1,92 +1,68 @@
 package hmda.validation.dsl
 
+import scala.language.implicitConversions
 import scala.util.Try
 
-trait CommonDsl {
-  implicit class Subject[T](data: T) {
-    private def test(predicate: Predicate[T]): Result = {
-      predicate.validate(data) match {
-        case true => Success()
-        case false => Failure(predicate.failure)
-      }
-    }
-
-    private def testNot(predicate: Predicate[T]): Result = {
-      predicate.validate(data) match {
-        case true => Failure(predicate.failure)
-        case false => Success()
-      }
-    }
-
-    def is(predicate: Predicate[T]): Result = {
-      test(predicate)
-    }
-
-    def not(predicate: Predicate[T]): Result = {
-      testNot(predicate)
-    }
-
-  }
-
-  def equalTo[T](that: T): Predicate[T] = new Predicate[T] {
+object PredicateDefaults {
```

Comment:
Yes, I'd like `PredicateCommon` or `CommonPredicates`; `PredicateInstances` is also fine though a little vague.  (I suppose "common" is also vague, but it expresses an intent, even if not super-precise, so I prefer it.)